core/base/inc/Rtypes.h

```diff
@@ -284,67 +284,75 @@ private:          \
          return ::ROOT::Internal::THashConsistencyHolder<decltype(*this)>::fgHashConsistency;                   \
       }                                                                                                         \
       return false; /* unreacheable */                                                                          \
-   }                                                                                                            \
+   } /** \endcond */                                                                                            \
                                                                                                                 \
 public:                                                                                                         \
-   static Version_t Class_Version() { return id; }                                                              \
-   virtual_keyword TClass *IsA() const overrd { return name::Class(); }                                         \
-   virtual_keyword void ShowMembers(TMemberInspector &insp) const overrd                                        \
+   /** \return Version of this class */ static Version_t Class_Version() { return id; }                         \
+   /** \see ::Class() */ virtual_keyword TClass *IsA() const overrd { return name::Class(); }                   \
+   /** \cond HIDDEN_SYMBOLS */ virtual_keyword void ShowMembers(TMemberInspector &insp) const overrd            \
    {                                                                                                            \
       ::ROOT::Class_ShowMembers(name::Class(), this, insp);                                                     \
-   }                                                                                                            \
-   void StreamerNVirtual(TBuffer &ClassDef_StreamerNVirtual_b) { name::Streamer(ClassDef_StreamerNVirtual_b); } \
-   static const char *DeclFileName() { return __FILE__; }
-
-#define _ClassDefOutline_(name,id, virtual_keyword, overrd) \
-   _ClassDefBase_(name,id, virtual_keyword, overrd)       \
-private: \
-   static atomic_TClass_ptr fgIsA; \
-public: \
-   static int ImplFileLine(); \
-   static const char *ImplFileName(); \
-   static const char *Class_Name(); \
-   static TClass *Dictionary(); \
-   static TClass *Class(); \
+   } /** \endcond */                                                                                            \
+   /** \cond HIDDEN_SYMBOLS */ void StreamerNVirtual(TBuffer &ClassDef_StreamerNVirtual_b)                      \
```

Comment:
Actually I think this ought to be shown.  This can (actually is) used in some user written custom streamer.